BYT/View Controllers/FoaasViewController.swift

```diff
@@ -41,6 +41,10 @@ class FoaasViewController: UIViewController, FoaasViewDelegate, FoaasSettingMenu
         makeRequest()
         updateSettingsMenu()
         
+        UIView.animate(withDuration: 2.0) {
```

Comment:
2 seconds is way too long from a UX perspective. Shorten this to something like  <= 0.3